tips-and-tricks/uv_install.md

```diff
@@ -0,0 +1,60 @@
+# UV Install on HPC
+
+UV is a new Python package manager which offers a few helpful features.
+[Website is here](https://docs.astral.sh/uv/)
+
+This short guide doesn't explain how to use UV generally but just how to get UV to work out of a directory, which is what you would want if you wanted the Python version stored in the project space.
+
+## Installation
+
+To install UV, run the following commands.
+It should be done in the project directory if you want all the users on the project to access it.
+
+`cd /go/to/project/directory`
+
+```bash
+mkdir uv_install
+mkdir uv_install/bin
+mkdir uv_install/pythons
+mkdir uv_install/python_bins
+mkdir uv_install/cache
+mkdir uv_install/tools
+
+curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="${PWD}/uv_install/bin" INSTALLER_NO_MODIFY_PATH=1 sh
```

Comment:
This is quite a long line, but only just overspills GitHub's standard display width. Do you think it would make sense to split it? I'm honestly not sure myself.

```suggestion
curl -LsSf https://astral.sh/uv/install.sh | \
    env UV_INSTALL_DIR="${PWD}/uv_install/bin" INSTALLER_NO_MODIFY_PATH=1 sh
```